python/ray/data/tests/test_all_to_all.py

```diff
@@ -1392,15 +1392,18 @@ def test_groupby_arrow_multi_agg(
         for agg in ["sum", "min", "max", "mean", "std", "quantile"]
     }
 
-    def _round_to_14_digits(row):
+    def _round_to_13_digits(row):
         return {
             # NOTE: Pandas and Arrow diverge on 14th digit (due to different formula
             #       used with diverging FP numerical stability), hence we round it up
```

Comment:
Is this guaranteed by the pandas or PyArrow interfaces? If not, this test might break if we upgrade either library. Might be worth using a more forgiving tolerance

Reply:
Not guaranteed. We want to keep it tight, and can revisit once that happens.